Makefile

```
@@ -45,6 +45,10 @@ configure-database-roles:
 	    --set=db_read_password=${db_read_password} \
 	    -f configure-database-roles.sql
 
+pypgstac-load:
+	PGHOST=${db_host} PGPORT=5432 PGDATABASE=postgres PGUSER=postgres PGPASSWORD=${db_admin_password} \
+	    pypgstac load items ${ndjson_file} --method upsert
```

We'll want to set `load items` vs `load collections` if we're using the same make command to load both data types:
```suggestion
	    pypgstac load ${type} ${ndjson_file} --method upsert
```

https://github.com/stac-utils/pgstac/blob/main/pypgstac/pypgstac/pypgstac.py#L70